2996_reflection/reflection.md

```diff
@@ -4060,7 +4085,7 @@ consteval bool is_const(info r);
 consteval bool is_volatile(info r);
 ```
 
-[#]{.pnum} *Returns*: `true` if `r` designates a const or volatile type (respectively), a const- or volatile-qualified member function type (respectively), or an object or function with such a type. Otherwise, `false`.
+[#]{.pnum} *Returns*: `true` if `r` designates a const or volatile type (respectively), a const- or volatile-qualified member function type (respectively), or an object, variable, or function with such a type. Otherwise, `false`.
```

Comment:
```cpp
struct C { int const x; };
```
Does `is_const(^C::x)` evaluate to `true` or `false`?

Reply:
Yeah, let's make that work.

Reply:
Also "member function types" aren't a thing, so we should just say "const- or volatile-qualified function type" (which ought to capture abominable functions).